packages/menu/src/SubMenu.tsx

```diff
@@ -44,8 +45,9 @@ import {
 } from './styles';
 import { Size } from './types';
 
-const SubMenuContainerClassName = createUniqueClassName('sub-menu-container');
-const IconButtonClassName = createUniqueClassName('icon-button');
+const subMenuContainerClassName = createUniqueClassName('sub-menu-container');
```

Comment:
non blocking but does it make more sense to define these in the styles files?

Reply:
This component isn't using the new file structure so when we make that switch we can move it. Right now all the styles are in this same file.